business-wiki-frontend/shared/components/PageDetails.tsx

```diff
@@ -0,0 +1,13 @@
+import { Page } from "../models/Page";
+
+const PageDetails = (props: { value: Page }) => {
```

Comment:
Destrukturyzacja propsów.

Reply:
Resolved in 6ff644a78a749be16d784b2d2c59d2a93f20ad90